docker/test/integration/minifi/validators/MultiFileOutputValidator.py

```diff
@@ -56,7 +56,10 @@ def validate(self):
             self.file_timestamps[full_path] = os.path.getmtime(full_path)
             logging.info("New file added %s", full_path)
 
-            if len(self.file_timestamps) == self.expected_file_count:
-                return self.check_expected_content(full_dir)
+        if self.expected_file_count != 0 and len(self.file_timestamps) != self.expected_file_count:
```

Comment:
Can you add this new behavior to the documentation string, please?

Reply:
Sure, updated the description in 2e445dfc7c600e5ac4d72bf7a6096b1220af84fe